tests/cold/test_ca_crt_migration.py

```
@@ -0,0 +1,156 @@
+            fixture_store=fixture_store,
+            plan=self.plan_resource,
+            target_namespace=target_namespace,
+        )
+
+    def test_check_vms(
+        self,
+        prepared_plan,
+        ca_crt_source_provider,
+        destination_provider,
+        source_provider_data,
+        target_namespace,
```

Ruff (0.15.17)

[warning] Unused method argument: `target_namespace`

(ARG002)